src/routes/messages/subagent-marker.ts

```
@@ -9,57 +10,137 @@ export interface SubagentMarker {
-const parseSubagentMarkerFromSystemReminder = (
-  text: string,
+export const parseSubagentMarkerFromFirstUser = (
+  payload: AnthropicMessagesPayload,
 ): SubagentMarker | null => {
+  const inspection = inspectSubagentMarkerFromFirstUser(payload)
+  return inspection.kind === "valid" ? inspection.marker : null
+}
+
+const extractMarkerPayloadFromReminderLine = (
+  line: string,
+): string | null => {
```

ESLint

[error] Replace `⏎··line:·string,⏎` with `line:·string`

(prettier/prettier)